docs/reference/upgrade/disable-shard-alloc.asciidoc

```diff
@@ -1,9 +1,9 @@
 
-When you shut down a node, the allocation process waits for
+When you shut down a data node, the allocation process waits for
 `index.unassigned.node_left.delayed_timeout` (by default, one minute) before
 starting to replicate the shards on that node to other nodes in the cluster,
 which can involve a lot of I/O. Since the node is shortly going to be
-restarted, this I/O is unnecessary. You can avoid racing the clock by 
+restarted, this I/O is unnecessary. You can avoid racing the clock by
 <<cluster-routing-allocation-enable,disabling allocation>> of replicas before
 shutting down the node:
```

Comment:
Rather than saying it's necessary for restarting a data node, I think we can change the scope of this recommendation like this:

```suggestion
shutting down data nodes:
```

Reply:
++, thanks.